appdev/guides/writeable-dirs.rst

```diff
@@ -0,0 +1,90 @@
+Writable directories
+====================
+
+App confinement is part of the Ubuntu Touch security concept. Data can be
+exchanged between apps only according to the AppArmor policies. This being said,
+apps can only read and write files that are located in one of three app 
+specific directories explained in this guide.
+
+.. Note::
+    The app can write debug messages into the app log file located at 
+    ``/home/phablet/.cache/upstart/application-click-<fullappname>_<appname>_<version>.log``.
+    To write into the log file, use the Qt debug functions.
+
+Config
+^^^^^^
```

Comment:
RST is interesting in that many special characters are allowed for use in headers, and any character it finds will be used as the next heading level down.

By using `^^^` before `---`, the `^^^` became second-level headings and `---` third-level headings. This means that `StandardPaths in C++` and so on are subsections of` App data`. See [Headings in the documentation contribute section](https://docs.ubports.com/en/latest/contribute/documentation.html#headings) to find appropriate characters for each heading level.

Reply:
Done